.pre-commit-config.yaml

```diff
@@ -64,3 +66,17 @@ repos:
     rev: "v4.0.0-alpha.8"
     hooks:
       - id: prettier
+        name: prettier
+        entry: npx prettier task_manager --write **/*.html --ignore-unknown
+        language: node
+        additional_dependencies: ["prettier@4.0.0-alpha.8"]
```

Comment:
possible way to go is to make it a system hook and install node, prettier, and plugin on your own 
you're doing almost the same stuff now as plugin is installed locally

Reply:
or smth like [this](https://adamj.eu/tech/2023/02/09/pre-commit-hooks-unsupported-tools/#node-hook-stylelint-a-css-linter) (I like it more)